aten/src/ATen/native/quantized/cpu/qclamp.cpp

```diff
@@ -128,30 +128,21 @@ Tensor clamp_quantized_cpu(
   return qy;
 }
 
-// hardtanh is clamp with default min==-1.0f and default max==1.0f
-Tensor hardtanh_quantized_cpu(
+Tensor& clamp_out_quantized_cpu(
     const Tensor& qx,
-    const Scalar& min,
-    const Scalar& max) {
-  Tensor qy;
-  qy = quantized_clamp_impl(qx, min, max);
-  return qy;
-}
-
-Tensor& hardtanh_out_quantized_cpu(const Tensor& qx,
-    const Scalar& min,
-    const Scalar& max,
+    const optional<Scalar>& min,
+    const optional<Scalar>& max,
     Tensor& result) {
-  result = quantized_clamp_impl(qx, min, max);
+  result = clamp_quantized_cpu(qx, min, max);
   return result;
 }
 
-Tensor& hardtanh_quantized_cpu_(
+Tensor& clamp_quantized_cpu_(
     Tensor& self,
-    const Scalar& min,
-    const Scalar& max) {
+    const optional<Scalar>& min,
+    const optional<Scalar>& max) {
   Tensor qy;
-  qy = quantized_clamp_impl(self, min, max);
+  qy = clamp_quantized_cpu(self, min, max);
   // This can be optimized in a future PR if it becomes a bottleneck.
```

Comment:
This comment makes me nervous. I also found comment like https://github.com/pytorch/pytorch/blob/04fe4382ec8e6031d353530d6b8446ac9c9f507e/torch/csrc/jit/passes/quantization/helper.cpp#L169. I hope I am not opening a can of worms here, LOL

Reply:
That commented also added 1.5 years ago 😬

Yeah, it looks like the current state is that out of these 4 ops: `hardtanh`, `hardtanh_`, `clamp`, `clamp_`, quantized currently supports all but the last one. And technically, this PR will add support for `clamp_`.

I think... we can count on CI to alert us if there's a really good reason for not adding support for that 4th variant?